packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -193,24 +205,136 @@ const PersonalDetailsForm = ({
                             data-testid='last_name'
                         />
                     )}
-                    {!is_appstore && !is_qualified_for_idv && !is_rendered_for_onfido && (
+                    {!is_qualified_for_idv && !is_rendered_for_onfido && !is_qualified_for_poa && (
                         <FormSubHeader title={localize('Other details')} />
                     )}
                     {'date_of_birth' in values && (
                         <DateOfBirthField
                             name='date_of_birth'
-                            required={is_svg || is_appstore}
+                            required={is_svg}
                             label={getNameAndDobLabels().dob_label}
                             hint={getFieldHint(localize('date of birth'))}
                             disabled={
                                 isFieldImmutable('date_of_birth', editable_fields) ||
                                 (values?.date_of_birth && has_real_account)
                             }
                             placeholder={localize('01-07-1999')}
-                            portal_id={is_appstore ? '' : 'modal_root'}
+                            portal_id='modal_root'
                             data_testid='date_of_birth'
                         />
                     )}
+                    {'address_line_1' in values && (
+                        <FormInputField
+                            name='address_line_1'
+                            label={localize('First line of address*')}
+                            disabled={isFieldImmutable('address_line_1', editable_fields)}
+                            data-testid='address_line_1'
+                            autoComplete='off'
+                            data-lpignore='true'
+                            type='text'
+                            maxLength={70}
+                            required
+                            onChange={handleChange}
+                            onBlur={handleBlur}
+                            error={touched.address_line_1 && errors.address_line_1}
+                            value={values.address_line_1}
```

Comment:
No need to pass `onChange`, `onBlur` & `value` as these are being handled by `FormInputField` itself

Reply:
Will add a TODO and handle this refactor in a planned card